examples/getstarted/config/server.js

```diff
@@ -0,0 +1,7 @@
+module.exports = ({ env }) => ({
+  host: env('HOST', 'localhost'),
```

Comment:
Why not set it to `0.0.0.0` instead of `localhost` (in order to match [server.js](packages/strapi-generate-new/lib/resources/files/config/server.js))

Reply:
Why not yes